R/count_teleconnections.R

```diff
@@ -133,6 +137,21 @@ count_watershed_teleconnections <- function(data_dir,
 
         distance_values %>% mean()/m_to_km -> avg_withdr_dis_km
 
+        # Get number of cities
+        watershed_usage_table %>%
+          subset(Watershed_DVSN_ID %in% city_intake_ids) -> city_usage
+        if(nrow(city_usage) == 0){
```

Comment:
When would the number of rows ever be zero? The algorithm isn't dealing with any groundwater cases at this point.

Reply:
@swd-turner There are some cities where their watershed is not being used by any other cities. Those watersheds are not included in the external water usage table and so when you do this filter for a city that doesn't have any other dependent cities, it will not have any DVSN_IDs in the table.